src/bootstrap/src/utils/proc_macro_deps.rs

```diff
@@ -47,6 +48,7 @@ pub static CRATES: &[&str] = &[
     "stable_deref_trait",
     "strsim",
     "syn",
+    "synom",
```

Comment:
This isn't needed any more, is it?

I don't see `synom` anywhere in Cargo.lock.

Reply:
`x.py test tidy --bless` fixed it. :)